src/extensiveFormulation.jl

```diff
@@ -53,12 +53,16 @@ function extensive_formulation(model, param)
             end
         end
     end
-
-    #Instantiate the problem creating dynamic constraint at each node
-    for t = 1 : (T)
+    #Add state constraints
+    for t = 1 :(T+1)
```

Comment:
We should let T here and put T = model.stageNumber at the beginning of the file. Having a T+1 does not have really a meaning